.github/workflows/docker.yaml

```diff
@@ -11,6 +11,7 @@ jobs:
         pgpool_version:
           - 4.5.0
           - 4.4.5
+          - 4.4.9
```

Comment:
(a) This isn't sufficient: you'll need to update `README.md`, `UPGRADE.md`, `charts/pgpool-cloudsql/values.schema.json` and `charts/pgpool-cloudsql/values.yaml` -- grepping for `4.4.5` will show all.

(b) as long as we're here, we may as well deal with CVE-2024-45624 and make the latest versions in all of the main release channels available: 4.5.4, 4.4.9, 4.3.12, 4.2.19 and 4.1.22

Reply:
👍